src/Apps/Order/Components/ItemReview.tsx

```diff
@@ -75,7 +73,14 @@ export const ItemReview: React.FC<ItemReviewProps> = ({
           {shortDescription}
         </Text>
       </Flex>
-      <Image maxHeight={375} width={185} src={url} alt={title} />
+      {image && (
```

Comment:
does it make sense to check `image.resized` property before we render `Image`? 🤔

Reply:
I was thinking if image.resized was null then image would be as well? I can add it in just in to be safe though.

Reply:
you might be right, I'm not sure if it's possible to have image without resized. adding the check for extra safety sounds good 👍